README.md

```diff
@@ -33,6 +33,19 @@ Cards:
   }
 }
 ```
+```shared/items.lua
```

Comment:
That's not bow that works, you use this after the code block to specify language for syntax highlighting